src/test/java/LoginTests.java

```diff
@@ -1,26 +1,16 @@
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebDriver;
-import org.openqa.selenium.WebElement;
-import org.openqa.selenium.chrome.ChromeDriver;
-import org.openqa.selenium.chrome.ChromeOptions;
-import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
 import org.testng.annotations.Test;
+import Pages.LoginPage;
 
-import java.time.Duration;
-
-public class LoginTests extends BaseTest {
+public class LoginTests extends BaseTest{
+    LoginPage loginPage;
+    @BeforeMethod
+    void setup(){
+        loginPage= new LoginPage(getThreadDriver());
+        getThreadDriver().get(url);
+    }
     @Test
-    public void loginEmptyEmailPassword() {
-//      Added ChromeOptions argument below to fix websocket error
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--remote-allow-origins=*");
-
-        WebDriver driver = new ChromeDriver(options);
-        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
-
-        String url = "https://qa.koel.app/";
-        driver.get(url);
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-        driver.quit();
+    public void validLoginPassword(){
+        loginPage.loginCorrectCred();
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example

![Clean_Test](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/104761482/77566b1c-b789-4cc9-9223-d369449ecba8)